### ISSUE TITLE
Adicionar Macad|3D ?

### ISSUE THREAD

**Issue by jorgepz** [yorikvanhavre/BIMLivre#1]
I've never used the program but it seems a good tool to add to the list, ainda que nao esteja em portugues 😆  https://macad3d.net/

**Comment by yorikvanhavre** [comment]
Yes, I saw it too some time ago... I'd like first to see better if it is any good for BIM, though. Unfortunately it's windows only :sweat_smile:

**Comment by jorgepz** [comment]
Oh, yes. Didn't see that. If anyone tries Macad3D out it could be posted here, or in other discussion forum designed for it ...
